code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -390,3 +390,95 @@
 		whistle.whistler = null
 		whistle = null
 	return ..()
+
+/obj/item/runic_vendor_staff
+	name = "staff of runic vendormancy"
+	desc = "This staff allows you to summon, detonate and throw Runic Vendors. A modern spin on the old Geomancy with 3 summoning charges that can be recovered with a simple magical channeling."
+	icon_state = "vendor_staff"
+	inhand_icon_state = "vendor_staff"
+	lefthand_file = 'icons/mob/inhands/weapons/staves_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
+	icon = 'icons/obj/weapons/guns/magic.dmi'
+	slot_flags = ITEM_SLOT_BACK
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 18
+	damtype = BURN
```

Comment:
Why does this do burn damage?

Reply:
It was just a placeholder copy of lava staff I left behind, but fixed now.